src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java

```diff
@@ -44,6 +44,9 @@
  * <a href="https://www.w3schools.com/html/html_blocks.asp">HTML block-tag</a>,
  * nested paragraph tags are allowed to do that.</li>
  * </ul>
+ * <p>
+ * <b>ATTENTION:</b> This Check ignores HTML comments.
```

Comment:
we need to add something about "if P nested, no violations at all, no any violations."
see https://github.com/checkstyle/checkstyle/pull/15686/files#r1787687046

Reply:
done

https://github.com/checkstyle/checkstyle/blob/cb7cac3b2e167e9fc260dc64576c87ab45257cfd/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java#L48-L53